### PR TITLE
Add asserter inheritance map

### LINKED ISSUE #248: Add a "map" of asserters hierarchy
As asserters inherit their parent's assertions, i think we could add a kind of hierarchy map to show what asserter inherits what other.

Thoughts ?

### PR DESCRIPTION
**Pull request by mikaelrandy**
Fix #248

Reword asserters documentation introduction
Add an inheritance map for asserters as a tree

### CODE REVIEW

File: source/en/asserters.rst [hunk 1]
```diff
@@ -10,8 +10,47 @@ Asserters collection
...
+To write more explicit and less wordy tests, atoum provide several asserters who give access to specific assertions related to tested var.
+
+As atoum different asserters are specializations of manipulated items, asserters inherits from asserters they specialize.
+It help keep consistency between asserters and force to use same assertion names.
+
+This is the asserters inheritance tree:
+
+.. code-block:: shell
+
+    -- asserter (abstract)
```
Comment (Grummfy) [inline source/en/asserters.rst]: A lot are missing... so we can add '...' or all of them.

Comment (mikaelrandy) [inline source/en/asserters.rst]: I only add asserters already documented.
The only exception is about `afterDestructionOf` that i havent found in the code.

Comment (Grummfy) [inline source/en/asserters.rst]: but there is more documented ....

Comment (mikaelrandy) [inline source/en/asserters.rst]: Like ?

Comment (Grummfy) [inline source/en/asserters.rst]: forget, ... each time I add one, I find it ;)

**Pull request merged**